packages/ai-chat/src/common/chat-service.ts

```diff
@@ -169,7 +169,7 @@ export class ChatServiceImpl implements ChatService {
 
     deleteSession(sessionId: string): void {
         const sessionIndex = this._sessions.findIndex(candidate => candidate.id === sessionId);
-        if (~sessionIndex) { return; }
+        if (!~sessionIndex) { return; }
```

Comment:
Can we refactor this to not use the `~` operator?
```suggestion
        if (sessionIndex === -1) { return; }
```

Reply:
Absolutly!!